script4.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+	<head>
+	<script type='text/javascript'>
+		var start = {
+			'2015-07-07': [
+				{ totalItemsSold: 20, totalPrice: 100 },
+				{ totalItemsSold: 5, totalPrice: 205 },
+				{ totalItemsSold: 10, totalPrice: 20 }
+			],
+			'2015-08-08': [
+				{ totalItemsSold: 5, totalPrice: 10 },
+				{ totalItemsSold: 10, totalPrice: 200 },
+				{ totalItemsSold: 12, totalPrice: 10 }
+			],
+			'2015-09-09': [
+				{ totalItemsSold: 90, totalPrice: 300 },
+				{ totalItemsSold: 23, totalPrice: 15 },
+				{ totalItemsSold: 12, totalPrice: 100 }
+			]
+		};
+
+		function computeObject(obj) {
+			var result = [[null], ['totalItemsSold'], ['totalPrice']]; //puts the title values in the array
```

Comment:
i commited the file again with a fix! the 'null' is still hardcoded, since it isn't reflected anywhere in the object - the rest is dynamic now, hope it fits